komposable-architecture/src/main/java/com/toggl/komposable/internal/MutableStateFlowStore.kt

```diff
@@ -61,19 +61,18 @@ internal class MutableStateFlowStore<State, Action : Any> private constructor(
             lateinit var send: (List<Action>) -> Unit
             send = { actions ->
                 storeScope.launch(context = dispatcherProvider.main) {
-                    var tempState = state.value
-                    val mutableValue = Mutable({ tempState }) { tempState = it }
-
-                    val effects = actions.flatMap { action ->
+                    val result: ReduceResult<State, Action> = actions.fold(ReduceResult(state.value, noEffect())) { accResult, action ->
```

Comment:
NIT: Can we cache the `noEffect()` to avoid constantly getting a new one on every `send`?